package.json

```diff
@@ -21,12 +21,12 @@
     "remark-validate-links": "^11.0.2"
   },
   "scripts": {
-    "start": "gatsby build && gatsby serve",
-    "start:prefix": "gatsby build --prefix-paths && gatsby serve --prefix-paths",
-    "dev": "gatsby develop",
-    "dev:https": "gatsby develop --https --host localhost.corp.adobe.com --port 9000",
-    "build": "gatsby build",
-    "serve": "gatsby serve",
+    "start": "NODE_OPTIONS='--max-old-space-size=8192' gatsby build && gatsby serve",
+    "start:prefix": "NODE_OPTIONS='--max-old-space-size=8192' gatsby build --prefix-paths && gatsby serve --prefix-paths",
+    "dev": "NODE_OPTIONS='--max-old-space-size=8192' gatsby develop",
```

Comment:
Related to internal ticket COMDOX-860

I still get a JS memory error on `yarn dev`:

```
<--- Last few GCs --->

[8386:0x7fca18008000]   219648 ms: Mark-sweep (reduce) 7926.3 (8216.2) -> 7926.3 (8211.0) MB, 329.8 / 0.1 ms  (average mu = 0.115, current mu = 0.000) last resort GC in old space requested
[8386:0x7fca18008000]   219992 ms: Mark-sweep 7945.2 (8229.8) -> 7945.1 (8230.8) MB, 301.0 / 0.1 ms  (average mu = 0.121, current mu = 0.126) allocation failure GC in old space requested


<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
...
```

It looks like the `NODE_OPTIONS` env var is being passed to the command as expected based on the memory value in the stack trace. I'm not sure why it's not enough.

Reply:
I get a JS memory error on yarn dev too but I couldn't find out the reason at this time. At least it fixed yarn start because it also started to fail with memory error.

Reply:
I'm running an [optimization tool](https://www.npmjs.com/package/imageoptim-cli) on the `src/pages/_images` directory and will retry `yarn dev` after to see if it helps.

Most images cannot be optimized any further, but some images are being optimized up to ~50%.

Reply:
It still fails after using the [imageoptim-cli](https://www.npmjs.com/package/imageoptim-cli) tool. See attachment for optimization log.

[imageoptim-cli.log](https://github.com/AdobeDocs/commerce-frontend-core/files/13663314/imageoptim-cli.log)

Reply:
As long as the GitHub Actions work (deploy & validation), we can merge to unblock PRs.

I can open a separate ticket to investigate the develop runtime issues. Your call @dshevtsov.

Reply:
Yes, I plan to continue my research of the yarn dev issue later.